### ISSUE TITLE
No role is defined for "Click here to view full recipe" control.

### ISSUE THREAD

**Issue by SuryacharanM** [dotnet/maui#6473]
GitHubTags:
#.NET MAUI; #A11ySev2; #A11yMAS; #AILimited; #A11yTCS; #Android; #AndroidPhone; #WCAG4.1.2; #GH_.NETMAUI_Android_Apr2022; #Name Role Value​;

### Description

Talkback is not announcing the role for the "Click here to view full recipe" control.

Similar issue observed in the following scenarios:
1. .NET MAUI<<My Recipes<<New: No role is defined for "New" control.
2. .NET MAUI<<My Recipes<<Edit: No role is defined for "Edit" control.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
3. Swipe till "search box" and enter recipe name and search.
4. "Search result" page will open.
5. Swipe till any recipe and double tap.
6. "Recipe" Screen will open.
7. Swipe till "Click here to view full recipe" control and observe the issue.

https://user-images.githubusercontent.com/86710788/165446749-7ca32a9d-31cc-4185-a016-dacb23ae396b.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Relevant log output

```shell
Talkback should announce the role for the "Click here to view full recipe" control i.e., "Click here to view full recipe", button. Tap to activate.
```

**Comment by keerthiduvvuri** [comment]
Hi @rachelkang i have verified this issue in ".NET MAUI" App, Version 1.0.0
The issue is reproducing still. Can you please provide us version details in which these are fixed. 
Thanks.

**Comment by rachelkang** [comment]
Hi, @keerthiduvvuri - please let us know your feedback after you've tested again against the latest version! It should be fixed. Thank you

**Comment by keerthiduvvuri** [comment]
Hi @rachelkang, I have verified this bug. The issue is Fixed.
I cannot close the bug, because this bug was logged by another tester. Can you close the bug from you'r end.

https://user-images.githubusercontent.com/98728920/186145055-578f32d1-242f-443d-96bf-d157ebcec61b.mp4
